### PR TITLE
create new organization

### PR DESCRIPTION
**Pull request by UnsungHero97**
create org button
Screen Shot 2019-10-14 at 4 30 06 PM
create org modal
Screen Shot 2019-10-14 at 4 30 30 PM
create org pending
Screen Shot 2019-10-14 at 4 31 07 PM
create org failure
Screen Shot 2019-10-14 at 4 32 55 PM

### CODE REVIEW

File: src/containers/orgs/OrgsContainer.js [hunk 1]
```diff
@@ -132,27 +156,80 @@ class OrgsContainer extends Component<Props> {
...
+
+    this.setState({ isVisibleNewOrgModal: true, newOrgTitle: '' });
   }
 
   renderOrgCard = (org :Map) => {
 
+    const orgId :UUID = org.get('id');
     let description :string = org.get('description', '');
 
     // TODO: refactor as a utility function
```
Comment (solomontang) [inline src/containers/orgs/OrgsContainer.js]: Looks like you're trying to show a snippet of text truncated by ellipses at the end. Consider `--webkit-line-clamp` for multi line blocks that end in ellipses.

https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp

File: src/utils/ValidationUtils.js [hunk 2]
```diff
@@ -33,17 +33,17 @@ function validateNonEmptyArray(value :any[], validatorFn :ValidatorFn) :boolean
   return true;
 }
 
-function isValidUUID(value :any) :boolean {
+function isValidUUID(value :any) :boolean %checks {
```
Comment (solomontang) [inline src/utils/ValidationUtils.js]: TIL

File: src/containers/orgs/OrgsContainer.js (hunk 1 above)
Comment (UnsungHero97) [reply src/containers/orgs/OrgsContainer.js]: 😍

Comment (UnsungHero97) [reply src/containers/orgs/OrgsContainer.js]: Done!

**Pull request merged**